ladybug_rhino/bakegeometry.py

```diff
@@ -137,11 +137,25 @@ def _get_attributes(layer_name=None, attributes=None):
 
 
 def _get_layer(layer_name):
-    """Get a layer index from the Rhino document from the ladyer name."""
+    """Get a layer index from the Rhino document from the layer name."""
     layer_table = doc.Layers  # layer table
-    layer_index = layer_table.Find(layer_name, True)
+    layer_index = layer_table.FindByFullPath(layer_name, True)
+    combined_part_names = []
     if layer_index < 0:
-        parent_layer = docobj.Layer()
-        parent_layer.Name = layer_name
-        layer_index = layer_table.Add(parent_layer)
+        part_names = layer_name.split("::") #  full name could be layer::sublayer::sublayer
+        for i in range(len(part_names)):
+            combined_part_names.append("::".join(part_names[0:i+1])) #  [0] = layer ,  [1] = layer::sublayer, [2] = layer::sublayer::sublayer
```

Comment:
Can ew make sure that all of the lines of code here obey [our hard character limit of 89 characters](https://github.com/ladybug-tools/contributing/wiki/python-style-guide#maximum-line-length)?